### ISSUE TITLE
Card Scanner Text

### ISSUE THREAD

**Issue by jpgSouza** [nateshmbhat/card-scanner-flutter#47]
Its possible to change the scan message? At the moment the message is "Scan your card to proceed".

**Comment by costarf** [comment]
Yes, and that's on Android, because on iOS is "Scan the back of your Credit Card to proceed"...

Any news on how to change this?

**Comment by Isuru-Nanayakkara** [comment]
I'm looking to do this too.
